cmd/node/main.go

```diff
@@ -546,6 +564,20 @@ func startNode(ctx *cli.Context, log *logger.Logger) error {
 	return nil
 }
 
+func getPrometheusJoinURL(serversConfigurationFileName string) (string, error) {
+	serversConfig, err := core.LoadServersPConfig(serversConfigurationFileName)
+	if err != nil {
+		return "", err
+	}
+	joinURL := serversConfig.Prometheus.PrometheusJoinURL
+	statusURL := strings.Replace(joinURL, "/join", "/status", 1)
```

Comment:
See comment below, regarding `PrometheusJoinURL`. This is one of the situations why it's better to have a base url, otherwise we will get into situations like this where we need to do a lot of tricks to build a correct url

Reply:
done